src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageBuildAction.java

```diff
@@ -255,7 +224,7 @@ public CoverageStatistics getStatistics() {
      */
     @SuppressWarnings("unused") // Called by jelly view
     public List<Baseline> getBaselines() {
-        return List.of(Baseline.PROJECT, Baseline.MODIFIED_FILES, Baseline.MODIFIED_LINES);
+        return List.of(Baseline.PROJECT, Baseline.MODIFIED_LINES);
```

Comment:
Only overall project baseline is needed on the summary page. The changes to this method conflicts with [cicd\_474 && cicd\_490 by dtang-parasoft · Pull Request #56 · jenkinsci/parasoft-findings-plugin](https://github.com/jenkinsci/parasoft-findings-plugin/pull/56/files#top)